icecap/ci/run.sh

```diff
@@ -1,5 +1,31 @@
+#!/bin/bash
+#
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT AND LICENSING
+#
+# See the `LICENSE_MIT.markdown` file in the Veracruz root directory
+# for licensing and copyright information.
+
 set -e
 
+error_df()
+{
+    set +e
+    echo
+    echo df -H
+    df -H
+    echo
+    echo du -sh
+    du -sh /nix /root /work
+    echo
+    echo du -k
+    du -k /work | sort -rn | head -n 30
+}
+trap error_df EXIT
+
```

Comment:
A fragment like this could be useful for debugging, but are we sure it belongs in `main`? I feel like it has an obfuscating effect. Perhaps we could get in the habit of temporarily inserting this into PRs when debugging them as necessary.

Reply:
I've removed this fragment. It was already a separate PR anyway: https://github.com/veracruz-project/veracruz/pull/314